deepinv/physics/multiscale.py

```diff
@@ -0,0 +1,137 @@
+import torch
+
+from deepinv.physics import Physics, LinearPhysics
+from deepinv.physics.blur import Upsampling
+
+
+class MultiScalePhysics(Physics):
```

Comment:
I feel like `MultiScalePhysics` sounds like a base class when it is a wrapper class. Maybe `PhysicsMultiScaler`?